receiver/sqlserverreceiver/scraper.go

```diff
@@ -400,6 +401,14 @@ func (s *sqlServerScraperHelper) recordDatabasePerfCounterMetrics(ctx context.Co
 			} else {
 				s.mb.RecordSqlserverDeadlockRateDataPoint(now, val.(float64))
 			}
+		case pageLifeExpectancy:
+			val, err := retrieveInt(row, valueKey)
+			if err != nil {
+				err = fmt.Errorf("failed to parse valueKey for row %d: %w in %s", i, err, pageLifeExpectancy)
+				errs = append(errs, err)
+			} else {
+				s.mb.RecordSqlserverPageLifeExpectancyDataPoint(now, val.(int64))
```

Comment:
I don't have a Windows machine to test this change. Should we remove the invocation of `RecordSqlserverPageLifeExpectancyDataPoint` in `recorder.go` or somewhere to avoid duplicate reports on Windows?

Reply:
Thanks for calling this out, @sincejune, it looks like we're double reporting on Windows. I wasn't able to 100% confirm but following the logic I think you're right. I've filed #40104 to address this. This issue is not unique to this metric, but all metrics that are available both via perf counters and querying SQL Server instances. I think it's safe to move forward with this change and address this as part of the bug, but let me know if you disagree.

Reply:
I’m okay with addressing the issue separately.